shell/platform/windows/flutter_windows_view.cc

```diff
@@ -105,6 +105,16 @@ void FlutterWindowsView::ForceRedraw() {
   }
 }
 
+#ifndef WINUWP
```

Comment:
If we have a `SetCompositionMode` style API, then that should be indirected through `WindowBindingHandler` to the respective win32 or winuwp window. rather than ifdef'ing here.  Each underlying impl can figure out if the requested mode is supported and configure, return true else false.  For Win32 it would succeed if composition mode is DirectComposition and OS version >= win8.  In the future we can also succeed for WinUIComp once the angle parts land with a version check.  For Win32 mode, only WinUIComp mode would succeed.  It would also be good to have a prefereddefault method on the binding handler to allow each platform to express a preference for the default for the case when setcompositionmode isn't called.  for win32 that can default to none initially and we can then flip it to DirectComposition at the point we're happy with the stability.

We should also verify that linking changes don't break win7.. will need to loadlibrary / getprocaddress and dcomp calls

Reply:
Angle already does that. It loads the direct composition calls through loadlibrary. we could do the same to determine whether dcomp si supported so that API can return true if supported.